.claude/commands/commit.md

```
@@ -204,17 +212,15 @@ coerces via a safe NaN/inf check before Int conversion.
 
 Closes #774
 
-🤖 Autogenerated by Claude (AI-native maintenance).
-Co-Authored-By: Claude <noreply@anthropic.com>
+🤖 AI-assisted maintenance.
 ```
 
 **Dep bump:**
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)